pkgs/applications/video/simplescreenrecorder/default.nix

```diff
@@ -24,8 +24,8 @@ stdenv.mkDerivation rec {
     done
```

Comment:
The `sed` call is no longer necessary https://github.com/MaartenBaert/ssr/commit/853863a04c58539cc6d2a4ee680753d8618aa5a6